src/main/java/org/prebid/server/bidder/conversant/ConversantBidder.java

```diff
@@ -115,7 +114,7 @@ private ExtImpConversant parseImpExt(Imp imp, int impIndex) {
             throw new PreBidException(String.format("Impression[%d] missing ext.bidder object", impIndex));
         }
 
-        if (StringUtils.isBlank(extImp.getSiteId())) {
+        if (StringUtils.isEmpty(extImp.getSiteId())) {
```

Comment:
not necessary. In go the don't have null

Reply:
What if `siteId` is whitespace?
Frankly, `isBlank` should be used only if we really need it but in this case it will make additional logic.